### PR TITLE
Spelling fixes, clarification, better language

### PR DESCRIPTION
**Pull request by comradekingu**
"Agreement License" spotted by @rafaelff at Weblate :)

### CODE REVIEW

**Comment by nijel** [comment]
Thanks for the suggestions, it seems right, however any change done in the terms of the service should be followed by getting new agreement from the users and doing that for this minor change would be annoying. Therefore I will postpone merging of this until we have some other change in the TOS which would trigger requesting new agreement from users.

File: wlhosted/templates/legal/documents/tos.html [hunk 1]
```diff
@@ -82,44 +82,44 @@ <h2>{% blocktrans %}Liability for Damage{% endblocktrans %}</h2>
...
+<p class="item">{% blocktrans %}Not specifying translation license means that the translations are available under same license as the given Project itself.{% endblocktrans %}</p>
 
 <p class="item">{% blocktrans %}Should the Project opt in for the Translation Memory, license to use the translation is granted to all Translation Memory users.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}The User agrees to the license Project has specified upon contributing to the Project.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The User agrees, upon contributing to a Project, to the license the Project has specified.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}The User agrees to use his name and email as authorship in the VCS commits. The User understands that this grant is non revocable due to nature of the VCS.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The User agrees to use of own name and email as authorship in the VCS commits. The User understands that this grant is non revocable due to nature of the VCS.{% endblocktrans %}</p>
```
Comment (comradekingu) [inline wlhosted/templates/legal/documents/tos.html]: of VCS systems?

File: wlhosted/templates/legal/documents/tos.html [hunk 2]
```diff
@@ -82,44 +82,44 @@ <h2>{% blocktrans %}Liability for Damage{% endblocktrans %}</h2>
...
 
 <h2>{% blocktrans %}Translations{% endblocktrans %}</h2>
 
-<p class="item">{% blocktrans %}The Service organizes translation into individual Projects, where the Owner is responsible for managing them and for specifying accurately the Project license.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The Service organizes translation into individual Projects, where the Owner is responsible for managing them and for specifying accurately the license of the Project.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}Not specifying translation license means that the translations are available under same license as project itself.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}Not specifying translation license means that the translations are available under same license as the given Project itself.{% endblocktrans %}</p>
 
 <p class="item">{% blocktrans %}Should the Project opt in for the Translation Memory, license to use the translation is granted to all Translation Memory users.{% endblocktrans %}</p>
```
Comment (comradekingu) [inline wlhosted/templates/legal/documents/tos.html]: Does the translation memory license extend to translation house and other related services?

File: wlhosted/templates/legal/documents/privacy.html [hunk 3]
```diff
@@ -27,7 +27,7 @@ <h2>{% blocktrans %}Personal Data we process{% endblocktrans %}</h2>
 <dd>{% trans "Additionally, email is used for notification of watched events" %}</dd>
 
 <dt>{% trans "Password in hashed form" %}</dt>
-<dd>{% trans "Used to authenticate user if configured" %}</dt>
+<dd>{% trans "Used to authenticate user, if configured" %}</dt>
```
Comment (comradekingu) [inline wlhosted/templates/legal/documents/privacy.html]: "User"?

Comment (comradekingu) [inline wlhosted/templates/legal/documents/privacy.html]:
```suggestion
<dd>{% trans "Used to authenticate the User, if configured" %}</dt>
```

File: wlhosted/templates/legal/documents/tos.html (hunk 1 above)
Comment (comradekingu) [inline wlhosted/templates/legal/documents/tos.html]:
```suggestion
<p class="item">{% blocktrans %}The User agrees to use of own name and email as authorship in the VCS commits. The User understands that this grant is non revocable due to nature of the VCS systems.{% endblocktrans %}</p>
```

File: wlhosted/templates/legal/documents/privacy.html [hunk 4]
```diff
@@ -59,19 +59,19 @@ <h2>{% blocktrans %}Access to the Personal Data{% endblocktrans %}</h2>
...
-<p>{% blocktrans %}The Personal Data is stored in the Service until User deletes his account in the service.{% endblocktrans %}</p>
+<p>{% blocktrans %}The Personal Data is stored in the Service until the User deletes their account on the service.{% endblocktrans %}</p>
 
-<p>{% blocktrans %}Access log information might be collected for longer period for the purpose of establishing, exercise or defence of legal claims.{% endblocktrans %}</p>
+<p>{% blocktrans %}Access log information might be collected for a longer period for the purpose of establishing, exercising or defending legal claims.{% endblocktrans %}</p>
 
 <h2>{% blocktrans %}Your rights{% endblocktrans %}</h2>
 
-<p>{% blocktrans %}User provides use Personal Data voluntarily. Without this we are not able to provide our services.{% endblocktrans %}</p>
+<p>{% blocktrans %}The User provides use of Personal Data voluntarily. Without this Weblate is not able to provide our services.{% endblocktrans %}</p>
```
Comment (comradekingu) [inline wlhosted/templates/legal/documents/privacy.html]: Either "The user provides User Personal Data", or "of".
```suggestion
<p>{% blocktrans %}The User provides use of Personal Data voluntarily. Without this Personal Data Weblate is not able to provide our services.{% endblocktrans %}</p>
```

**Comment by comradekingu** [comment]
@nijel I agree, somewhat, though I mainly tried to stay with non-structural changes, and not accidentally open loopholes. A versioned system where one can read the diff would be great. I will probably find more fixes. This is what I do for a living now, so my familiarity with GDPR is increasing.

File: wlhosted/templates/legal/documents/tos.html [hunk 5]
```diff
@@ -82,44 +82,44 @@ <h2>{% blocktrans %}Liability for Damage{% endblocktrans %}</h2>
...
 
-<p class="item">{% blocktrans %}If the User directly and effectively informs the Provider that he no longer wishes to receive business information from the Provider, the Provider shall refrain from sending further unsolicited business communications.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}If the User directly and effectively informs the Provider of no longer wishing to receive business information from the Provider, the Provider shall refrain from sending further unsolicited business communications.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}The User declares that he/she is aware of his/her rights under Article 12 and Article 21 of the Personal Data Protection Act and declares that all the Personal Data are accurate and true and have been provided voluntarily.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The User declares awareness of own rights under Article 12 and Article 21 of the Personal Data Protection Act and declares that all the Personal Data is accurate and true and has been provided voluntarily.{% endblocktrans %}</p>
```
Comment (comradekingu) [inline wlhosted/templates/legal/documents/tos.html]: Following a debate with ssantos at hosted Weblate,
There could be links to 
http://www.privacy-regulation.eu/en/article-21-right-to-object-GDPR.htm  
and
http://www.privacy-regulation.eu/en/article-12-transparent-information-communication-and-modalities-for-the-exercise-of-the-rights-of-the-data-subject-GDPR.htm
I don't know that these are the best ones, but a start.

File: wlhosted/templates/legal/documents/privacy.html [hunk 6]
```diff
@@ -12,12 +12,12 @@ <h2>{% blocktrans %}Personal Data Controller{% endblocktrans %}</h2>
...
 </p>
 
-<p>{% blocktrans with privacy_contact="privacy@weblate.org" %}We have appointed a data protection officer who may be reached via {{privacy_contact}}.{% endblocktrans %}</p>
+<p>{% blocktrans with privacy_contact="privacy@weblate.org" %}Weblate has appointed a data protection officer who may be reached via {{privacy_contact}}.{% endblocktrans %}</p>
 
 <h2>{% blocktrans %}Personal Data we process{% endblocktrans %}</h2>
 
 <p>
-{% trans "We process only Personal Data User provides us by using our service:" %}
+{% trans "Weblate only process Personal Data the User provides us by using our service:" %}
```
Comment (comradekingu) [inline wlhosted/templates/legal/documents/privacy.html]: - [ ] Sort out clarifications of whether EU-GDPR sections are the same for all.

Though this is modeled on the EU-GDPR, which means citizens in the EU eco zone, it applies to all?

**Comment by nijel** [comment]
@orangesunny can you please review this change to terms of service and privacy policy? I originally though this should be included in some other update to those documents, but there is no update coming in near future, so better to review and merge this now...

**Comment by comradekingu** [comment]
@nijel is there anything in there about clicking on usernames and getting the e-mail copied, because I remember doing that, but not reading about it in the terms.

**Comment by nijel** [comment]
@comradekingu The user emails should not be anywhere in the web UI. In case they are it's clearly a bug.

**Comment by comradekingu** [comment]
@nijel I think they should be CAPTCHA-ed and requiring an account, in the UI. Having them in e-mails results in spam. https://github.com/WeblateOrg/weblate/issues/2701

**Comment by comradekingu** [comment]
Neverstale?

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**